css/index.css

```diff
@@ -143,4 +264,49 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/*about page*/
+.header-img {
+    margin: 0 10px;
+    padding-bottom: 30px;
+}
+
+.about {
+   border-bottom: 1px dashed black;  
+   padding-bottom: 30px;  
+   margin-left: 30px;
+}
+
+.main-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    line-height: 1.5;
+}
+
+.main-content .info-box {
+  padding: 20px;
+  border-bottom: 1px dashed black;
+  width: 365px;
+  margin: 20px 0;
+}
+
+.main-content .info-box h1 {
+
+}
+
+.main-content .info-box .box-img {
+    width: 320px;
+
+}
+
+.main-content .info-box .button {
+  padding: 10px 40px;
+  border: 1.5px solid black;
+  margin-top: 20px;
+}
+
+.end {
+    margin-bottom: 30px;
 }
```

Comment:
you can add cursor: pointer property to your buttons to indicate to the users that they are to be interacted with and are clickable. practice styling buttons a bit and you can get them to match the design file also